app/graphql/types/hmis_schema/project.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::Project < Types::BaseObject
```

Comment:
Did you generate this file with `rails g graphql:object GrdaWarehouse::Hud::Project`? I'm curious how much of these types we can generate

Reply:
I used the generators for all the files -- and then edited them to fit my needs (they don't consider that you might was multiple schemas for the controller for example, so I needed to move some things into multiple modules.) This file was generated, but was empty as I expect to add wrapper classes between all of these and the ActiveRecord objects as the schemas won't be the same)